README.md

```diff
@@ -49,86 +51,99 @@ on, we suggest opening an issue detailing your ideas first.
 
 Otherwise send us a pull request and we will be happy to review it.
 
-### Code Style
-
-All code should be formatted through `go fmt`. This tool will automatically
-format code for you. Although it is recommended you write code in this style
-and just use this tool to catch mistakes.
+#### 1. Install Dependencies
 
-### Building
+Install `go` and `base-devel` with _pacman_.
 
-Yay is easy to build with its only build dependency being `go` and the
-assumption of `base-devel` being installed.
+Note that _yay_ requires a few other projects (as vendored dependencies). These
+projects are stored in `vendor/`, are built into _yay_ at build time, and do not
+need to be installed separately:
```

Comment:
Looking good, I would add a warning to never edit files in the vendor directory though, you never know.

Reply:
OK, thanks for feedback. Added in a simple warning here.